src/include/parser/create_function_statement.h

```diff
@@ -0,0 +1,136 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+//create_function_statement.h
+//
+// Identification: src/include/parser/create_function_statement.h
+//
+// Copyright (c) 2015-16, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include "expression/abstract_expression.h"
+#include "type/types.h"
+#include "common/sql_node_visitor.h"
+#include "parser/sql_statement.h"
+
+namespace peloton {
+namespace parser {
+
+struct Parameter {
```

Comment:
It is better to use class instead of struct here because there are so many functions and variables in this struct. Use class you can use private and public but for struct all will be public here.